backend/src/models/v2/Schema.ts

```diff
@@ -14,7 +14,7 @@ export interface SchemaInterface {
   hidden: boolean
 
   kind: SchemaKindKeys
-  jsonSchema: unknown
+  jsonSchema: object
```

Comment:
Because the above interface is _bad_, you can probably use some types from 'json-schema' to help out (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/json-schema/index.d.ts)